content/docs/command-reference/gc.md

```diff
@@ -42,6 +42,9 @@ most cases removing data locally and in remote storage is irreversible.
 The default remote is cleaned (see `dvc config core.remote`) unless the
 `--remote` option is used.
 
+> Please note that `dvc gc` might try to pull `.dir` <abbr>cache</abbr> files
```

Comment:
- This note applies to the general case - consider moving it up
- May be we should change the language from "might" to "pulls"?
- Consider including a link to the https://dvc.org/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory - it contains information about `.dir` files.
- Consider being more explicit - it pulls them when they are missing and they are referenced in the commits GC analyzed in `.dvc` or `dvc.yaml` files.